### ISSUE TITLE
Collision with PSR-12 standards

### ISSUE THREAD

**Issue by celyes** [squizlabs/PHP_CodeSniffer#3012]
The library raises an error when private/protected properties of a class are not prefixed with an underscore, although PHP standard PSR-12 explicitly recommends not doing so. [See here](https://www.php-fig.org/psr/psr-12/#43-properties-and-constants)

any updates on this matter?

**Comment by jrfnl** [comment]
> The library raises an error when private/protected properties of a class are not prefixed with an underscore

@celyes It doesn't. Or rather: it does the opposite: it will warn when private/protected properties **_are_** prefixed with an underscore.

To proof my point:
```php
<?php

namespace Foo;

class Bar
{
    private $property;
    private $_prop;
}
```

If I run the PSR12 standard over that code, this is the result:
```
# phpcs -s ./phpcs-3012.php --standard=PSR12

FILE: phpcs-3012.php
-----------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-----------------------------------------------------------------------------------------------------------
 8 | WARNING | Property name "$_prop" should not be prefixed with an underscore to indicate visibility
   |         | (PSR2.Classes.PropertyDeclaration.Underscore)
-----------------------------------------------------------------------------------------------------------
```

I suspect you are not using the PSR12 ruleset, but some custom ruleset which is causing a sniff which _does_ demand the underscore prefixes to be triggered.

This is not a PHPCS PSR12 ruleset problem though.

**Comment by celyes** [comment]
Thank you! i'll check that
